sdcm/cluster_aws.py

```diff
@@ -143,9 +143,16 @@ def _create_on_demand_instances(self, count, interfaces, ec2_user_data, dc_idx=0
         instance_profile = self.instance_profile_name
         if instance_profile:
             params['IamInstanceProfile'] = {'Name': instance_profile}
+        ec2 = ec2_client.EC2ClientWrapper(region_name=self.region_names[dc_idx])
+        subnet_info = ec2.get_subnet_info(self._ec2_subnet_id[dc_idx])
+        region_name_with_az = subnet_info['AvailabilityZone']
+        LOGGER.info('Sending an On-Demand request with params: %s', params)
```

Comment:
the params has too much information, please put his print in debug level as well

Reply:
fixed